Chirp/Pages/ChatsPage/ChatsViewController.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  ChatsViewController.swift
+//  Chirp
+//
+//  Created by Ady on 6/7/23.
+//
+
+import UIKit
+
+class ChatsViewController: UIViewController {
+    
+    var chatsTableView: UITableView = {
+        let tableView = UITableView()
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.allowsSelection = true
+        tableView.backgroundColor = .systemBackground
+        return tableView
+    }()
+    
+    var searchController: UISearchController = {
+        let searchController = UISearchController(searchResultsController: nil)
+        searchController.searchBar.placeholder = "Search"
+        return searchController
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setupNavigationBarTitle()
+        setupSearchBar()
+        setupViews()
+        setupConstraints()
+    }
+    
+    private func setupNavigationBarTitle() {
+        title = "Chats"
+        navigationController?.navigationBar.prefersLargeTitles = true
+        // TODO: Check if this is not needed
+        navigationItem.hidesBackButton = true
+    }
+    
+    private func setupSearchBar() {
+        searchController.searchResultsUpdater = self
+        navigationItem.searchController = searchController
+    }
+    
+    private func setupViews() {
+        view.addSubview(chatsTableView)
+        chatsTableView.dataSource = self
+        chatsTableView.delegate = self
+        chatsTableView.register(ChatsTableViewCell.self, forCellReuseIdentifier: ChatsTableViewCell.identifier)
+    }
+    
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            chatsTableView.topAnchor.constraint(equalTo: view.topAnchor),
+            chatsTableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            chatsTableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            chatsTableView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+        ])
+    }
+}
+
+extension ChatsViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // TODO: Update when we have actual data
+        return 10
```

Comment:
This is great but can you connect it to a dummy datasource?
This will make it easier for us to plug and play when we start connecting it to the backend.

So, 
1. Create a struct called RecentChat.
2. Use the count here
3. Populate the cell with the data from the dummy datasource 
4. You can also install Kingfisher to help manage our images using the URL and set the placeholder to be the placeholder image